DS-285 - hide control panel in example stories (#317)
bifrost-vui/bifrost-front-end-library: CHANGELOG.mdx

<br/><br/>

## [Next Version](https://bifrost-front-end-library-vui.vercel.app/)

> ### Base
> - Hide the Storybook Control Panel on *Example* stories in Components and Sections. User may still activate Control Panel manually
> 

Reviewer: ```suggestion
### Documentation
- Storybook: Hide by default the Control Panel on *Example* stories.

```
